pkg/cloud/helpers.go

```diff
@@ -37,6 +38,12 @@ func setArrayIfNotEmpty(strArray []string, setFn setArray) {
 	}
 }
 
+func setIntIfNotZero(num int64, setFn setInt) {
```

Comment:
Maybe rename this to setIntIfGreaterThanZero.

Reply:
How about `setIfPositiveNumber`